README.md

```diff
@@ -4,17 +4,22 @@
 ## Prerequisites
 1. Install [Terraform](https://www.terraform.io/) version 0.38.7 or newer.
 2. Install [Terragrunt](https://terragrunt.gruntwork.io/) version v1.2.6 or newer.
-3. Install [Python](https://www.python.org/) version v3.10 or newer.
-4. Install [jsonpath-ng](https://pypi.org/project/jsonpath-ng/) version v1.5.3 or newer.
+3. Install [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl).
```

Comment:
Do we need kubectl installed locally? 🤔

Reply:
Fair point, for the deployment with python script it's not necessary.